src/main/java/chess/service/ChessService.java

```diff
@@ -0,0 +1,125 @@
+package chess.service;
+
+import chess.domain.ChessResult;
+import chess.domain.board.Board;
+import chess.domain.dao.BoardDao;
+import chess.domain.dao.PieceDao;
+import chess.domain.dto.BoardDto;
+import chess.domain.dto.PieceDto;
+import chess.domain.dto.request.MoveRequest;
+import chess.domain.dto.response.BoardAndPieceDto;
+import chess.domain.dto.response.Response;
+import chess.domain.dto.response.ResponseCode;
+import chess.domain.game.ChessGame;
+import chess.domain.piece.Piece;
+import chess.domain.piece.Team;
+import chess.domain.position.Position;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public final class ChessService {
+
+    private final PieceDao pieceDao;
+    private final BoardDao boardDao;
+    private ChessGame chessGame;
+
+    public ChessService() {
+        pieceDao = new PieceDao();
+        boardDao = new BoardDao();
+    }
+
+    public void chessBoardInit() {
+        try {
+            if (pieceDao.load() == null) {
+                pieceDao.save(new Board().unwrap());
+            }
```

Comment:
자바에서 null을 반환하다보면, NPE를 피하기위해 null check를 해주어야 해요
요러한 단점때문에 null 반환은 안티패턴이기도하고, 이것을 보완하기위해 나온것이 optional이죠ㅎㅎ

DAO에서 `Optional<Board>`를 반환하고 아래와 같이 사용해보시라고 코멘트를 남겼었습니다 ^_^
```suggestion
            pieceDao.load()
                    .orElseGet(() -> pieceDao.save(new Board().unwrap()));
```